### PR TITLE
Custom Jail Settings

### PR DESCRIPTION
**Pull request by Kimax89**
Everything has been tested and working.

### CODE REVIEW

**Comment by Kimax89** [comment]
Sorry @Jawshy

File: Altis_Life.Altis/core/civilian/fn_jailMe.sqf [hunk 1]
```diff
@@ -2,48 +2,54 @@
...
-private["_time","_bail","_esc","_countDown"];
+private["_time","_bail","_esc","_countDown","_jailtime","_badtime","_license"];
 
 params [
 	["_ret",[],[[]]],
 	["_bad",false,[false]]
 ];
 
+_jailtime = LIFE_SETTINGS(getNumber,"jail_time") * 60;
+_badtime = _jailtime * LIFE_SETTINGS(getNumber,"jail_disconnected") * 60;
```
Comment (BoGuu) [inline Altis_Life.Altis/core/civilian/fn_jailMe.sqf]: See you in 15 hours

https://i.gyazo.com/9b86d09f1e206be81b9f5a31a8b5f8a9.jpg

Comment (Kimax89) [inline Altis_Life.Altis/core/civilian/fn_jailMe.sqf]: Thats the first time i have seen that.

**Comment by BoGuu** [comment]
I think you missed the point.

`_badtime = (_jailtime * LIFE_SETTINGS(getNumber,"jail_disconnected")) * 60;`

where

`_jailtime = LIFE_SETTINGS(getNumber,"jail_time") * 60;`

**Comment by Kimax89** [comment]
I'm an idiot. :D

**Comment by danielstuart14** [comment]
working?

**Comment by BoGuu** [comment]
Don't pull this just yet, I have some qualms about certain things I've just spotted.

**Comment by BoGuu** [comment]
Conflicts, re go this

**Pull request closed without merging**